packages/twenty-server/src/engine/core-modules/auth/token/services/refresh-token.service.ts

```diff
@@ -90,6 +90,13 @@ export class RefreshTokenService {
       );
     }
 
+    if (!token.workspaceId) {
+      throw new AuthException(
+        'This refresh token is malformed',
+        AuthExceptionCode.INVALID_INPUT,
+      );
+    }
```

Comment:
This error allows the front to redirect to the login page when the token has no workspace.
Instead, it's a blocking error that forces the user to clean his browser storage.

Reply:
Users should all meet it when we will deploy. 
In 90d it will be useless.